packages/vite-plugin-cloudflare/src/constants.ts

```diff
@@ -1,9 +1,3 @@
-import type { Miniflare } from 'miniflare';
-
 export const ROUTER_WORKER_NAME = '__router-worker__';
 export const ASSET_WORKER_NAME = '__asset-worker__';
 export const ASSET_WORKERS_COMPATIBILITY_DATE = '2024-10-04';
```

Comment:
sorry it's unrelated to this PR, but `ASSET_WORKER_NAME` and `ASSET_WORKERS_COMPATIBILITY_DATE` are only used in the `miniflare-options.ts` file, maybe they could be simply moved there?